xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -276,10 +276,12 @@ unsigned int CActiveAEStream::AddData(uint8_t* const *data, unsigned int offset,
           if (m_format.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_TRUEHD)
           {
             m_currentBuffer->pkt->nb_samples += 2560;
-            uint8_t highByte = (minFrames >> 8) & 0xFF;
-            uint8_t lowByte = minFrames & 0xFF;
-            memcpy(m_currentBuffer->pkt->data[0]+m_currentBuffer->pkt->nb_samples-2, &highByte, 1);
-            memcpy(m_currentBuffer->pkt->data[0]+m_currentBuffer->pkt->nb_samples-1, &lowByte, 1);
+            // Copies the high byte and the low byte
```

Comment:
don't document the obvious. the variables are called low/high byte. this line of comments adds zero information